project/BillOfMaterialsPlugin.scala

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2019-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+import scala.xml.Elem
+
+import sbt.AutoPlugin
+import sbt.PluginTrigger
+import sbt.ProjectReference
+import sbt._
+
+object BillOfMaterialsPlugin extends AutoPlugin {
+
+  override def trigger = PluginTrigger.NoTrigger
+
+  object autoImport {
+    val includedProjects = settingKey[Seq[ProjectReference]]("")
+//    val fixedScalaVersion = settingKey[String]("")
+  }
+  import autoImport._
+  import sbt.Keys._
+  import sbt._
+
+  override def projectSettings: Seq[Def.Setting[_]] =
+    Seq(
+      // publish Maven Style
+      publishMavenStyle := true,
+      // Produce a single BOM with all the artifacts
+      crossVersion := CrossVersion.disabled, // this setting removes the scala bin version from the artifact name
+//      fixedScalaVersion := crossScalaVersions.value.head,
+//      crossScalaVersions := Seq(fixedScalaVersion.value),
+//      scalaVersion := fixedScalaVersion.value,
```

Comment:
There's a couple of things:

1. we should remove the commented lines (18, 30-32)
2. Since `crossVersion := CrossVersion.disabled` may not do what we expect (it may get overwritten), we could check the value and raise a warning or a failure telling the user they must add `crossVersion := CrossVersion.disabled` themselves (?)